frontend/public/components/cluster-settings/cluster-settings.jsx

```diff
@@ -1,55 +0,0 @@
-import * as React from 'react';
-import { Helmet } from 'react-helmet';
-
-import {PageHeading, DocumentationSidebar} from '../utils';
-import {LDAPSetting} from './ldap';
-import {CertsInfoContainer} from './certs-info';
-import {SafetyFirst} from '../safety-first';
-import {FLAGS, connectToFlags} from '../../features';
-import {TectonicChannel} from '../channel-operators/tectonic-channel';
-import {ContainerLinuxUpdates} from '../container-linux-update-operator/container-linux-updates';
-
-export const SettingsRow = ({children}) => <div className="row co-m-form-row">{children}</div>;
-export const SettingsLabel = ({children}) => <div className="col-sm-4 col-md-3"><strong>{children}</strong></div>;
-export const SettingsContent = ({children}) => <div className="col-sm-8 col-md-9">{children}</div>;
-
-export const ClusterSettingsPage = connectToFlags(FLAGS.CLUSTER_UPDATES)(
```

Comment:
We should remove this flag from features.ts.

Reply:
@spadgett Removed server and feature detection code.